app/LVM/userparameters/rabe.lvm.conf

```diff
@@ -0,0 +1,184 @@
+# the discovery rule.
+UserParameter=rabe.lvm.vgs.discovery,/usr/bin/sudo /sbin/vgs --noheadings --options vg_name,vg_uuid,vg_fmt | /usr/bin/awk 'BEGIN { printf "{\"data\":[" } { printf sep"{\"{#LVM_VG_NAME}\":\""$1"\","; printf "\"{#LVM_VG_UUID}\":\""$2"\","; printf "\"{#LVM_VG_TYPE}\":\""$3"\"}"; sep = "," } END { printf "]}" }'
+
+
+#
+# Discovery rule for getting a list of all available logical volumes (LV)
+#
+# Data gets parsed from the lvs command output and returned in the following
+# format:
+#
+# {
+#    "data" : [
+#       {
+#          "{#LVM_LV_FULL_NAME}" : "vg_example_01/lv_example_01"
+#          "{#LVM_LV_UUID}" : "dxi5Hw-Xo2m-SAVi-ypZh-QO4I-krpI-sR3N67",
+#          "{#LVM_LV_TYPE}" : "linear",
+#          "{#LVM_LV_ROLE}" : "public,origin,thickorigin",
+#          "{#LVM_LV_NAME}" : "lv_example_01"
+#          "{#LVM_LV_VG_NAME}" : "vg_example_01"
+#          "{#LVM_LV_PATH}" : "/dev/vg_example_01/lv_example_01",
+#       },
+#       {
+#          "{#LVM_LV_FULL_NAME}" : "vg_example_02/lv_example_02",
+#          "{#LVM_LV_UUID}" : "7rbSqn-Lnow-Xzpk-LdWf-w7ed-xGMQ-40qFd2",
+#          "{#LVM_LV_TYPE}" : "thin"
+#          "{#LVM_LV_ROLE}" : "public"
+#          "{#LVM_LV_NAME}" : "lv_example_02"
+#          "{#LVM_LV_VG_NAME}" : "vg_example_02"
+#          "{#LVM_LV_PATH}" : "/dev/vg_example_02/lv_example_02",
+#       },
+#       {
+#          "{#LVM_LV_FULL_NAME}" : "vg_example_02/lv_example_02_thin_pool_01",
+#          "{#LVM_LV_UUID}" : "UsCiDV-0B7p-krKP-1g1a-m17h-qo2J-E9xiTa",
+#          "{#LVM_LV_TYPE}" : "thin-pool",
+#          "{#LVM_LV_ROLE}" : "private",
+#          "{#LVM_LV_NAME}" : "lv_exmaple_02_thin_pool_01"
+#          "{#LVM_LV_VG_NAME}" : "vg_example_02"
+#          "{#LVM_LV_PATH}" : ""
+#       },
+#       {
+#          "{#LVM_LV_FULL_NAME}" : "vg_example_01/snap_lv_example_01"
+#          "{#LVM_LV_UUID}" : "nSdv5J-9Ovx-3A4S-ls5b-Ji6x-0fvQ-CeocLl",
+#          "{#LVM_LV_TYPE}" : "linear",
+#          "{#LVM_LV_ROLE}" : "public,snapshot,thicksnapshot",
+#          "{#LVM_LV_NAME}" : "snap_lv_example_01"
+#          "{#LVM_LV_VG_NAME}" : "vg_example_01"
+#          "{#LVM_LV_PATH}" : "/dev/vg_example_01/snap_lv_example_01",
+#       }
+#    ]
+# }
+#
+# Note, that the awk field parameters ($1 - $N) require double dollar sign
+# escaping ($$1) to prevent them from being interpreted as Zabbix positional
+# references.
+UserParameter=rabe.lvm.lvs.discovery[*],/usr/bin/sudo /sbin/lvs --noheadings --options lv_full_name,lv_uuid,segtype,lv_role,lv_name,vg_name,lv_path | /usr/bin/awk 'BEGIN { printf "{\"data\":[" } { printf sep"{\"{#LVM_LV_FULL_NAME}\":\""$$1"\","; printf "\"{#LVM_LV_UUID}\":\""$$2"\","; printf "\"{#LVM_LV_TYPE}\":\""$$3"\","; printf "\"{#LVM_LV_ROLE}\":\""$$4"\","; printf "\"{#LVM_LV_NAME}\":\""$$5"\","; printf "\"{#LVM_LV_VG_NAME}\":\""$$6"\","; printf "\"{#LVM_LV_PATH}\":\""$$7"\"}"; sep = "," } END { printf "]}" }'
```

Comment:
Ok th ecomments are different each time. This one does seem to be a discovery thing though. Wouldn't it need to be like the other discovery ones that say "do not require double dollar escaping"?